src/style.css

```
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  --blue: #00acdc;
+}
+
+.moyo__logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
+  padding-top: 10px;
+}
+
+.moyo__header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: list-item;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  align-items: center;
+  display: flex;
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: var(--blue);
+}
+
+.is-active {
+  text-decoration: none;
+  color: var(--blue);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 56px;
+  display: flex;
+  width: 37px;
```

and add position relative to .nav__link
```suggestion
  bottom: 0;
  display: flex;
  width: 100%;
```

---

> 

I'm corrected errors. Please check my work))